### ISSUE TITLE
checkbox well not selected when switch to DONE (light mode)

### ISSUE THREAD

**Issue by Geokeke** [cannibalox/logseq-dark-hpx#4]
When I finish something and click on the checkbox, the checkbox well not selected.

LATER:
![image](https://user-images.githubusercontent.com/22414088/112278616-ff029900-8cbd-11eb-9dbf-c53a42836475.png)
DONE:
![image](https://user-images.githubusercontent.com/22414088/112278707-10e43c00-8cbe-11eb-9077-25a2a9290178.png)

Thanks!

**Comment by cannibalox** [comment]
this should be fixed in v0.6.5 of logseq-darker-hpx.user.css  
there are a couple of other modificatrions, let me know if it works fine for you.

in case you have other tweaks and just need to add the fix without overwriting the whole file, search for the `checkbox` section and add:
```
.form-checkbox:checked {
		background-color: var(--ls-page-checkbox-border-color)   !important;
	}
```

**Comment by Geokeke** [comment]
v0.6.5 looks great! Thank you for making such a beautiful theme, I really enjoy it.
